src/ansys/fluent/core/workflow.py

```diff
@@ -988,6 +973,84 @@ def refresh_after_sleep(_):
             self.add_on_affected(refresh_after_sleep)
 
 
+class OldWorkflowWrapper:
+    """Wrap a Workflow object, adding methods to discover more about the relationships
+    between TaskObjects.
+
+    Methods
+    -------
+    __getattr__(attr)
+    __dir__()
+    __call__()
+    """
+
+    def __init__(self, workflow: PyMenuGeneric, command_source: PyMenuGeneric) -> None:
+        """Initialize WorkflowWrapper.
+
+        Parameters
+        ----------
+        workflow : PyMenuGeneric
+            The workflow object.
+        command_source : PyMenuGeneric
+            The application root for commanding.
+        """
+        self._workflow = workflow
+        self._command_source = command_source
+        self._python_task_names = []
+        self._lock = threading.RLock()
```

Comment:
What's this for?

Reply:
Thanks @seanpearsonuk, for pointing this out. It is not required in this context now. Will remove it.

Reply:
Same with other variables as well.